changelogs/fragments/5829-fix-yarn-global.yml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
+  - yarn - fix state=latest not working with global=true (https://github.com/ansible-collections/community.general/pull/5829).
+  - yarn - fix global=true to check for the configured global folder instead of assuming the default (https://github.com/ansible-collections/community.general/pull/5829)
+  - yarn - fix state=absent not working with global=true when the package does not include a binary (https://github.com/ansible-collections/community.general/pull/5829)
```

Comment:
```suggestion
  - yarn - fix ``state=latest`` not working with ``global=true`` (https://github.com/ansible-collections/community.general/pull/5829).
  - yarn - fix ``global=true`` to check for the configured global folder instead of assuming the default (https://github.com/ansible-collections/community.general/pull/5829)
  - yarn - fix ``state=absent`` not working with ``global=true`` when the package does not include a binary (https://github.com/ansible-collections/community.general/pull/5829)
```